appveyor.yml

```
@@ -3,10 +3,11 @@ build: off
 environment:
   global:
     SALT_FILEROOT: '%APPVEYOR_BUILD_FOLDER%'
-    SALT_STATE: mcafee-agent
-    SALT_URL: 'https://repo.saltstack.com/windows/Salt-Minion-3003.3-Py3-AMD64-Setup.exe'
+    SALT_PILLARROOT: '%APPVEYOR_BUILD_FOLDER%\tests\pillar\test-windows-main'
+    SALT_URL: 'https://repo.saltproject.io/windows/Salt-Minion-3005.1-5-Py3-AMD64-Setup.exe'
```

This version of salt is causing the latest problem. Let's go back to 3004.2....

```suggestion
    SALT_URL: https://repo.saltproject.io/windows/Salt-Minion-3004.2-1-Py3-AMD64-Setup.exe
```